app/code/Magento/Catalog/Console/Command/ImagesResizeCommand.php

```diff
@@ -18,31 +22,31 @@ class ImagesResizeCommand extends \Symfony\Component\Console\Command\Command
     protected $productCollectionFactory;
 
     /**
-     * @var \Magento\Catalog\Api\ProductRepositoryInterface
+     * @var QueueFactory
      */
-    protected $productRepository;
+    protected $queueFactory;
 
     /**
-     * @var \Magento\Catalog\Model\Product\Image\CacheFactory
+     * @var Options
      */
-    protected $imageCacheFactory;
+    protected $options;
 
     /**
      * @param \Magento\Framework\App\State $appState
      * @param \Magento\Catalog\Model\ResourceModel\Product\CollectionFactory $productCollectionFactory
-     * @param \Magento\Catalog\Api\ProductRepositoryInterface $productRepository
-     * @param \Magento\Catalog\Model\Product\Image\CacheFactory $imageCacheFactory
+     * @param QueueFactory $queueFactory
+     * @param Options $options
      */
     public function __construct(
         \Magento\Framework\App\State $appState,
         \Magento\Catalog\Model\ResourceModel\Product\CollectionFactory $productCollectionFactory,
-        \Magento\Catalog\Api\ProductRepositoryInterface $productRepository,
-        \Magento\Catalog\Model\Product\Image\CacheFactory $imageCacheFactory
+        QueueFactory $queueFactory,
+        Options $options
     ) {
         $this->appState = $appState;
         $this->productCollectionFactory = $productCollectionFactory;
-        $this->productRepository = $productRepository;
-        $this->imageCacheFactory = $imageCacheFactory;
+        $this->queueFactory = $queueFactory;
```

Comment:
Due to backwards compatibility police it prohibited to add new non-optional arguments to the constructor. Please make it optional and nullable and get the required dependency with the ObjectManager static call if value is null.